### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by FrancoisSalmon — DISMISSED** [review]
conflicts fix ok for me

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -9,6 +9,18 @@ import FirstTeam from "@pages/FirstTeam";
...
+import BergersDesLouveteaux from "@pages/BergersDesLouveteaux";
+import Arbitre from "@pages/Arbitre";
+import Classement from "@pages/Classement";
+import Philosophie from "@pages/Philosophie";
+import Effectif from "@pages/Effectif";
+import VieQuotidienne from "@pages/VieQuotidienne";
+import Formation from "@pages/Formation";
+import Candidature from "@pages/Candidature";
+import Ambition from "@pages/Ambition";
+import Partenaire from "@pages/Partenaire";
```
Comment (LoicBrassart) [inline frontend/src/App.jsx:23]: Name your components in English

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Won't read, big refactoring to do :(

**Review by FrancoisSalmon — DISMISSED** [review]
no more french, ok for me

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Warning name styled components

File: frontend/src/pages/Partner/index.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import SPartenaire from "./style";
+
+export default function Partenaire() {
+  return (
+    <SPartenaire>
```
[the remark this answers is not in the document]
Comment (FrancoisSalmon) [reply frontend/src/pages/Partner/index.jsx:5]: still in french

**Review by FrancoisSalmon — CHANGES_REQUESTED** [review]
still french in partner pages

**Review by FrancoisSalmon — DISMISSED** [review]
the final i hope

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict

**Review by FrancoisSalmon — DISMISSED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/Archi/Footer/index.jsx [hunk 3]
```diff
@@ -1,8 +1,9 @@
-import imgGriffe from "@assets/Griffebleu.png";
-import imgBTZ from "@assets/BTZ.png";
-import imgSAH from "@assets/Logo_SAH.png";
-import imgLFH from "@assets/FFHB.png";
+
```
Comment (LoicBrassart) [inline frontend/src/components/Archi/Footer/index.jsx:1]: Why add an empty line ? You edited this on GHub, didn't you ?

File: frontend/src/components/Archi/Header/index.jsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
-import Navbar from "@components/Archi/Navbar";
-import LogoDesktop from "@assets/Pictures/LogoDesktop.svg";
-import LogoMobile from "@assets/Pictures/LogoMobile.png";
 import { Link } from "react-router-dom";
+import LogoDesktop from "@assets/images/logos/sah-wolf-triangle.svg";
+import LogoMobile from "@assets/images/logos/clubs/sah.png";
+import Navbar from "../Navbar";
```
Comment (LoicBrassart) [inline frontend/src/components/Archi/Header/index.jsx:4]: Use aliases

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please fix these

**Review by FrancoisSalmon — APPROVED** [review]
corrections ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
NmE! <3

**Pull request merged**